docs/release/release_0_2_7.rst

```diff
@@ -0,0 +1,81 @@
+Announcement: napari 0.2.7
+==========================
+
+We're happy to announce the release of napari v0.2.7!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-base
+rendering), and the scientific Python stack (numpy, scipy).
+
+
```

Comment:
I would add a highlights section here. Here's the PRs that I think would be particularly exciting from a user's perspective:

- threshold slider for much improved isosurface rendering
- dockable widgets (!)
- slice information on sliders
- dramatically improved performance with many invisible layers

Reply:
added a highlight section with these and the play button!